bench_state.go

```diff
@@ -79,13 +80,18 @@ func (s *benchmarkState) recordLatency(d time.Duration) {
 	s.statter.Timing("latency", d)
 }
 
-func (s *benchmarkState) printLatencies(out output) {
-	// TODO JSON output?
+// Returns a mapping of quantiles to latency values
+func (s *benchmarkState) getLatencies(out output) map[string]string {
```

Comment:
should we use `map[float64]time.Duration` so that the quantile conversion doesn't lose the type/precision?